### DISCUSSION TITLE
Binder support for astra toolbox

### DISCUSSION THREAD

**Discussion by kolibril13** [astra-toolbox/astra-toolbox#393] [Ideas]
It might be nice to try the astra toolbox online, without installation.
Here a proof of concept that this works for a small example:
![image](https://user-images.githubusercontent.com/44469195/111076771-02dc3180-84ee-11eb-8bbd-b4e400fa74fe.png)
and here you can try this out for yourself: 
[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/kolibril13/astra-toolbox/HEAD?filepath=samples%2Fpython%2Fbinder_example.ipynb)
https://mybinder.org/v2/gh/kolibril13/astra-toolbox/HEAD?filepath=samples%2Fpython%2Fbinder_example.ipynb

Here are the steps to reproduce the creation:
* New anaconda environment with python3.6
* conda install -c astra-toolbox astra-toolbox
* conda env export > environment.yml

EDIT: The time until the toolbox is installed in binder varies a bit: I tried it 3 times, and it was between 10 and 90 seconds.

**Comment by meetaig** [comment]
See my comment from your other submitted issue
